### ISSUE TITLE
Contact page UI implementation

### ISSUE THREAD

**Issue by Masud-00** [ByteMonk-GCECT/GCECT-APP#53]
## Issue Type:
UI related

## Description:
Address head line should be in center
EditText view should be improved

## ScreenShots / Video:

https://user-images.githubusercontent.com/70429079/161413995-06996369-2560-48b8-bb33-2a9fa1d1e4cb.mp4

**Comment by Masud-00** [comment]
I would like to work on this issue. Please assign me

**Comment by HeySitam** [comment]
yes bro and also try to place it at the center of the joining line

**Comment by Parth-Pitrubhakta** [comment]
I would like to work on this issue

**Comment by Masud-00** [comment]
I already fixed this issue.But Unfortunately by mistake I remove myself from the assignment .please assign me again .so I can create pull request.

**Comment by priyanshu942** [comment]
Please assign me this issue under GSSOC'23.
